### PR TITLE
VER-81595: Remove null rowCount field from result set metadata

### PR DESCRIPTION
**Pull request by jonathanl-bq**
After some discussion during stand-up, we decided to remove the rowCount field from the Result Set entirely. Unlike Postgres, the rowCount currently isn't returned by Vertica. Other client drivers, such as JDBC, also do not return the row count, nor do they display it in any way. Since having the null rowCount is somewhat misleading, we decided the best option for now would be to remove and document this change. We may come back to support this later.

### CODE REVIEW

**Review by alexr-bq — APPROVED** [review]

File: packages/vertica-nodejs/lib/result.js [hunk 1]
```diff
@@ -35,13 +34,10 @@ class Result {
     }
     if (match) {
       this.command = match[1]
+      // [VERTICA specific] Unlike Postgres, rowCount is not sent back by Vertica, but if it ever is supported, it could be set here
       if (match[3]) {
```
Comment (jeremyp-bq) [inline packages/vertica-nodejs/lib/result.js:38]: Does this still get 3 matches from Vertica, or should it be 2?

Comment (jonathanl-bq) [reply packages/vertica-nodejs/lib/result.js:38]: For most queries, the tag is blank completely, so there's no match. Certain queries like COMMIT just have "COMMIT" as the tag, but that's it.

**Review by DMickens — COMMENTED** [review]
I've got some concerns with this. Not necessarily with the changes that you made. I think you've made changes to accommodate how the driver handles command complete response, but I don't know that the driver is handling things correctly, or possibly we're just parsing unnecessarily. Going to look into this a bit more on my own and try and wrap my head around this.

File: packages/vertica-nodejs/lib/result.js (hunk 1 above)
Comment (jonathanl-bq) [reply packages/vertica-nodejs/lib/result.js:38]: > Furthermore, again assuming this is only handling a command complete message, ther are only 3 things that should be parsed out, meaning a populated array should have values at index 0, 1, and 2.

So I'm pretty sure that this is not how it parses the CommandComplete message. Note the regular expression used here. Looking at the Postgres documentation (https://www.postgresql.org/docs/current/protocol-message-formats.html) you can see that the tag can be in the form of "UPDATE rows" where rows is the row count as a string, which I believe is being parsed as an Int here.

In comparison, Wireshark shows that Vertica's Command Complete message has a blank Tag:
PostgreSQL
    Type: Command completion
    Length: 5
    Tag:

**Review by DMickens — CHANGES_REQUESTED** [review]
Jonathan helped explain what was happening here. In response to the conversation we decided the regex should be updated so that it isn't trying to match a row count. If there are any, updates to the matched array coming from the regex update should be handled.

**Review by jeremyp-bq — APPROVED** [review]

**Review by DMickens — APPROVED** [review]
All my concerns have been addressed, I think this is being handled in a good way now

**Pull request merged**